### ISSUE TITLE
Voxel size not read

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-io#82]
```
OMEZarrS3Opener.setLogging( true );
SpimData spimData = OMEZarrS3Opener.readURL("https://s3.embl.de/i2k-2020/ngff-example-data/v0.4/zyx.ome.zarr");

final String unit = spimData.getSequenceDescription().getViewSetupsOrdered().get( 0 ).getVoxelSize().unit();
final double[] voxelSize = spimData.getSequenceDescription().getViewSetupsOrdered().get( 0 ).getVoxelSize().dimensionsAsDoubleArray();

Prefs.showScaleBar(true); // for visual inspection
BdvFunctions.show( spimData ); // for visual inspection
```

yields "pixels" and `[1,1,1]` which I guess is wrong?

ping @constantinpape @K-Meech 

- [x] fix it
- [x] add a test for `unit` and `voxelSize` using above example

**Comment by K-Meech** [comment]
@tischi @KateMoreva - I think I already have a fix for this on my mobie-io branch (I needed it for converting the ome-zarr units / restricting units in the project creator). It's here: https://github.com/K-Meech/mobie-io/commit/46454fdd20f0ac72b97fb97de3642efc3feba89f I should be able to finish up this PR early next week (got busy with a paper!).

**Comment by constantinpape** [comment]
Please also add a test for this.

**Comment by constantinpape** [comment]
Hey @K-Meech, did you have a chance to look into this? It would be great to get this in soon to move on with the release.

**Comment by K-Meech** [comment]
Hi @constantinpape - not yet. I should have time tomorrow to look at this!

**Comment by tischi** [comment]
@K-Meech The issue was about _reading_ the voxel size and unit. Does your above PR about _writing_ OME-Zarr also address this?

**Comment by K-Meech** [comment]
Yes it does! I was working on writing the units + keeping it consistent with the project creator, when I noticed the reading issue. I fixed it alongside, as I needed it to read OME-ZARR units in the project creator.
I added a test in that PR too for the code at the top of this issue: https://github.com/mobie/mobie-io/blob/develop/src/test/java/spimdata/OmeZarrV4S3SpimDataTests.java#L15, and it passes now.
